third_party/3.6/click/decorators.pyi

```diff
@@ -29,6 +33,7 @@ def command(
     name: str = None,
     cls: type = Command,
     # Command
+    context_settings: Dict = None,
```

Comment:
Sounds like you should do at least `Optional[Dict[str, Any]]` then, or better yet, use `Mapping`. That Union wouldn't be the most complicated type in typeshed, but if you prefer not specifying the value type that's fine with me.

If you do though, you should almost certainly use Mapping rather than Dict, because Dict is invariant. Therefore, a `Dict[str, str]` wouldn't be compatible with an argument annotated as `Dict[str, Union[str, bool]]`.